NitroxServer/ConsoleCommands/PromoteCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NitroxModel.DataStructures.GameLogic;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Abstract.Type;
+
+namespace NitroxServer.ConsoleCommands
+{
+    internal class PromoteCommand : Command
+    {
+        public PromoteCommand() : base("promote", Perms.MODERATOR, "Sets specific permissions to a user")
```

Comment:
I feel like promote is the wrong name here as it could be a demotion if you set their permissions lower. Would giverole or something be more descriptive?